src/app/clusters/bindings/BindingManager.cpp

```diff
@@ -207,7 +207,7 @@ CHIP_ERROR BindingManager::NotifyBoundClusterChanged(EndpointId endpoint, Cluste
                 else
                 {
                     mPendingNotificationMap.AddPendingNotification(iter.GetIndex(), context);
```

Comment:
So if the device _is_ connected but `mBoundDeviceChangedHandler` is null this will add to `mPendingNotificationMap` and then do nothing?  Why is that useful?

Also, the multicast case assumes `mBoundDeviceChangedHandler` is non-null anyway, so we're being inconsistent about that null-check.

What is the actual behavior we want when `mBoundDeviceChangedHandler` is null, assuming that can happen?

Reply:
Added check to do nothing if `mBoundDeviceChangedHandler` is null.